src/client/app/effects/snippet.ts

```diff
@@ -288,6 +288,13 @@ export class SnippetEffects {
             case Snippet.ImportType.SAMPLE:
             case Snippet.ImportType.URL:
             case Snippet.ImportType.GIST:
+                let viewUrlMatch = new RegExp(`^${environment.current.config.editorUrl}\/#\/view`);
+                if (type === Snippet.ImportType.URL && viewUrlMatch.test(data)) {
+                    /* If importing from a view-mode URL, simply redirect the user */
```

Comment:
By "import" do you mean actually importing the snippet and opening it in editor mode? The view mode already does an import action but doesn't save the snippet to local storage. We now want the user to be able to paste in a "view-url", but now they can edit that snippet? I thought that was accomplished by the open in playground feature.